src/api/validators/validators.ts

```diff
@@ -0,0 +1,60 @@
+import { fetchEthValidatorTotalPerformance } from '../../utils/fetchEthValidatorData';
+import { fetchFtmValidatorTotalPerformance } from '../../utils/fetchFtmValidatorData';
+import { getKey, setKey } from '../../utils/cache';
+import { DailyBifiBuybackStats } from '../stats/bifibuyback/getBifiBuyback';
+
+export interface ValidatorPerformance {
+  eth: string | null;
+  ftm: string | null;
+  fuse: string | null;
+}
+
+const ETH_PERFORMANCE_KEY = 'VALIDATOR_PERFORMANCE';
+const INIT_DELAY = Number(process.env.BUYBACK_INIT_DELAY || 40 * 1000);
+const REFRESH_INTERVAL = 15 * 60 * 1000;
+
+const getEthValidatorPerformance = async () => {
+  const ethPerformance = await fetchEthValidatorTotalPerformance();
+  return ethPerformance.totalPerformanceEther ?? null;
+};
+
+const getFtmValidatorPerformance = async () => {
+  const ftmPerformance = await fetchFtmValidatorTotalPerformance();
+  return ftmPerformance.totalPerformanceEther ?? null;
+};
+
+const getFuseValidatorPerformance = async () => {
+  return null;
+};
+
+let validatorPerformance: ValidatorPerformance | undefined = undefined;
+
+const updateValidatorPerformance = async () => {
+  console.log('> updating validator performance');
+  try {
+    validatorPerformance = {
+      eth: await getEthValidatorPerformance(),
+      ftm: await getFtmValidatorPerformance(),
+      fuse: await getFuseValidatorPerformance(),
+    };
+    console.log('> updated validator performance');
+    saveToRedis();
+  } catch (e) {
+    console.error('> validator performance initialization failed', e);
+  }
```

Comment:
These 3 requests could be done concurrently so as to speed up the update. Right now `getFtmValidatorPerformance` doesn't start till `getEthValidatorPerformance` finishes.

This could look something like:
`const performances = await Promise.allSetled([getEth...(), getFtm...(), getFuse...()]);`
and assigning after this